src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -97,5 +102,74 @@ public static async Task<JsonResult> CreateBadRequestResult(List<ValidationError
 
             return await CreateResult(data, HttpStatusCode.BadRequest, JsonContentTypeApplicationJsonProblem);
         }
+
+        /// <summary>
+        /// Handles cache results.
+        /// </summary>
+        /// <typeparam name="TEntity">Entity type of results.</typeparam>
+        /// <param name="results">Results from cache get.</param>
+        /// <param name="logger">The logger.</param>
+        /// <returns>IActionResult.</returns>
+        public static async Task<ActionResult> HandleCacheResult<TEntity>(IEnumerable<TEntity> results, NgsaLog logger)
```

Comment:
Since we're no longer caching, we need rename the "HandleCache" and remove the "Cached Result" log information.  However, I'm currently thinking that we don't need them and we can just go with the InternalReturnOKResult() method.